compiler/rustc_driver/src/lib.rs

```diff
@@ -546,24 +546,12 @@ impl Compilation {
 #[derive(Copy, Clone)]
 pub struct RustcDefaultCalls;
 
-// FIXME remove these and use winapi 0.3 instead
-// Duplicates: bootstrap/compile.rs, librustc_errors/emitter.rs
```

Comment:
Can you also use `atty` in bootstrap and librustc_errors?

Reply:
I couldn't find the duplicates it's referring to. `rustc_errors` seems to use `atty` already, and I don't see anything in `bootstrap`. If you know of a spot that has a duplicate, I'm happy to update it!

Reply:
I found `library/test/src/helpers/isatty.rs`, but it might be tricky to convert that one since it's part of the runtime. This seems fine for now.

Reply:
Based on a Git blame, these functions were added way back in #42732. Initially that PR added the dependency `isatty`, but Mark-Simulacrum requested that it be removed in https://github.com/rust-lang/rust/pull/42732#discussion_r125191880 since it depended on something with long build times.